test/runtests.jl

```diff
@@ -205,6 +205,33 @@ function test_l1b_bool_mask_with_fake_data()
 
 end
 
+function test_mask_dilate()
+
+  mask = zeros(Int64, 5, 5)
+  mask[3, 3] = 1
+  kern = zeros(Int64, 3, 3) + 1
```

Comment:
The `ones` function should work here here, i.e. `ones(Int64, 3, 3)`. (There are also `trues` and `falses` functions that behave analogously, if you decide to switch to using booleans.)